charts/jaeger/values.yaml

```diff
@@ -199,40 +199,9 @@ storage:
       # cassandra.keyspace: jaeger_v1_test
       # cassandra.tls.enabled: "false"
   elasticsearch:
-    scheme: http
-    host: elasticsearch-master
-    port: 9200
-    anonymous: false
-    user: elastic
-    usePassword: true
-    password: changeme
-    # indexPrefix: test
-    ## Use existing secret (ignores previous password)
-    # existingSecret:
-    # existingSecretKey:
-    nodesWanOnly: false
-    extraEnv: []
-    ## ES related env vars to be configured on the concerned components
-    # - name: ES_SERVER_URLS
-    #   value: http://elasticsearch-master:9200
-    # - name: ES_USERNAME
-    #   value: elastic
-    # - name: ES_INDEX_PREFIX
-    #   value: test
-    ## ES related cmd line opts to be configured on the concerned components
-    cmdlineParams:
-      {}
-      # es.server-urls: http://elasticsearch-master:9200
-      # es.username: elastic
-      # es.index-prefix: test
-    tls:
-      enabled: false
-      secretName: es-tls-secret
-      # The mount properties of the secret
-      mountPath: /es-tls/ca-cert.pem
-      subPath: ca-cert.pem
-      # How ES_TLS_CA variable will be set in the various components
-      ca: /es-tls/ca-cert.pem
+    # Deprecated: use extraEnv
+    # indexPrefix: jaeger
+    # serverUrls: http://elasticsearch:9200
```

Comment:
in configmap we have things like these:
```
            {{ if or .Values.provisionDataStore.elasticsearch (eq .Values.storage.type "elasticsearch") }}
            {{- $es := .Values.storage.elasticsearch | default dict -}}
            {{- $scheme := $es.scheme | default "http" -}}
            {{- $port := $es.port | default 9200 -}}
            {{- $user := $es.user | default "elastic" -}}
            {{- $password := .Values.elasticsearch.secret.password | default "changeme" -}}
```
doesn't it mean they are trying to read values from this section?